openrtb_ext/imp_appnexus.go

```diff
@@ -1,20 +1,25 @@
 package openrtb_ext
 
-import "encoding/json"
+import (
+	"encoding/json"
+
+	"github.com/prebid/prebid-server/util/jsonutil"
+)
 
 // ExtImpAppnexus defines the contract for bidrequest.imp[i].ext.prebid.bidder.appnexus
 type ExtImpAppnexus struct {
-	LegacyPlacementId       int                     `json:"placementId"`
-	LegacyInvCode           string                  `json:"invCode"`
-	LegacyTrafficSourceCode string                  `json:"trafficSourceCode"`
-	PlacementId             int                     `json:"placement_id"`
-	InvCode                 string                  `json:"inv_code"`
-	Member                  string                  `json:"member"`
-	Keywords                []*ExtImpAppnexusKeyVal `json:"keywords"`
-	TrafficSourceCode       string                  `json:"traffic_source_code"`
-	Reserve                 float64                 `json:"reserve"`
-	Position                string                  `json:"position"`
-	UsePmtRule              *bool                   `json:"use_pmt_rule"`
+	DeprecatedPlacementId    jsonutil.StringInt      `json:"placementId"`
+	LegacyInvCode            string                  `json:"invCode"`
+	LegacyTrafficSourceCode  string                  `json:"trafficSourceCode"`
+	PlacementId              jsonutil.StringInt      `json:"placement_id"`
+	InvCode                  string                  `json:"inv_code"`
+	Member                   string                  `json:"member"`
+	Keywords                 []*ExtImpAppnexusKeyVal `json:"keywords"`
+	TrafficSourceCode        string                  `json:"traffic_source_code"`
+	Reserve                  float64                 `json:"reserve"`
+	Position                 string                  `json:"position"`
+	UsePmtRule               *bool                   `json:"use_pmt_rule"`
```

Comment:
Nitpick: Consider naming this variable `UsePaymentRule` to align with `DeprecatedUsePaymentRule `

Reply:
thank you! changed here - https://github.com/prebid/prebid-server/pull/2617/commits/310a5f8baee8655070551e78f473f70b7dad36ee